renovate.json

```
@@ -26,7 +26,7 @@
         "(^|/).+\\.yaml$"
       ],
       "matchStrings": [
-        "\\s*image:\\s+registry:\\s*[\\\"']?(?<registry>.*?)[\\\"']?\\s+(?:repository|name):\\s*[\\\"']?(?<repository>.*?)[\\\"']?\\s+tag:\\s*[\\\"']?(?<currentValue>[\\w+\\.\\-]*)(?:@(?<currentDigest>sha256:[a-f0-9]+))?[\\\"']?"
+        "[\\t ]*image:\\s+registry:\\s*[\\\"']?(?<registry>\\S+\\.\\S+?)[\\\"']?[\\t ]*\\r?\\n[\\t ]*(?:repository|name):\\s*[\\\"']?(?<repository>.*?)[\\\"']?[\\t ]*\\r?\\n[\\t ]*tag:\\s*[\\\"']?(?<currentValue>[\\w+\\.\\-]*)(?:@(?<currentDigest>sha256:[a-f0-9]+))?[\\\"']?",
```

The JSON linter doesn't like the last comma:
```suggestion
        "[\\t ]*image:\\s+registry:\\s*[\\\"']?(?<registry>\\S+\\.\\S+?)[\\\"']?[\\t ]*\\r?\\n[\\t ]*(?:repository|name):\\s*[\\\"']?(?<repository>.*?)[\\\"']?[\\t ]*\\r?\\n[\\t ]*tag:\\s*[\\\"']?(?<currentValue>[\\w+\\.\\-]*)(?:@(?<currentDigest>sha256:[a-f0-9]+))?[\\\"']?"
```

---

A, good one.